src/components/extensive/WorkdayCollapseGrid/WorkdayCollapseGrid.tsx

```diff
@@ -5,20 +5,29 @@ import { FC, useCallback, useMemo, useState } from "react";
 import { When } from "react-if";
 
 import Text from "@/components/elements/Text/Text";
+import { Framework, useFrameworkContext } from "@/context/framework.provider";
 
 import Icon, { IconNames } from "../Icon/Icon";
 import { useTableStatus } from "./hooks";
-import { Demographic, DEMOGRAPHIC_TYPES, DemographicType, WorkdayCollapseGridProps } from "./types";
+import {
+  Demographic,
+  DEMOGRAPHIC_TYPES,
+  DemographicType,
+  HBF_DEMOGRAPHIC_TYPES,
+  HBFDemographicType,
+  WorkdayCollapseGridProps
+} from "./types";
 import WorkdaySection from "./WorkdaySection";
 
 const WorkdayCollapseGrid: FC<WorkdayCollapseGridProps> = ({ title, demographics, variant, onChange }) => {
   const [open, setOpen] = useState(false);
   const t = useT();
-  const { total, status } = useTableStatus(demographics);
+  const { framework } = useFrameworkContext();
+  const { total, status } = useTableStatus(framework, demographics);
```

Comment:
There's no need to pass the framework in here. The `useTableStatus` hook can use `useFrameworkContext` internally.